documentation/staging/content/userguide/prerequisites/introduction.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Operator prerequisites"
+date: 2019-02-23T16:40:54-05:00
+weight: 1
+---
+
+
+For the current production release 3.2.3:
+
+* Kubernetes 1.16.15+, 1.17.13+, 1.18.10+, and 1.19.7+ (check with `kubectl version`).
+* Flannel networking v0.9.1-amd64 or later (check with `docker images | grep flannel`), Calico networking v3.16.1 or later,
+ *or* OpenShift SDN on OpenShift 4.3 systems.
+* Docker 18.9.1 or 19.03.1+ (check with `docker version`) *or* CRI-O 1.14.7+ (check with `crictl version | grep RuntimeVersion`).
+* Helm 3.3.4+ (check with `helm version --client --short`).
+* For domain home source type `Model in Image`, WebLogic Deploy Tooling 1.9.11.
+* Either Oracle WebLogic Server 12.2.1.3.0 with patch 29135930, Oracle WebLogic Server 12.2.1.4.0, or Oracle WebLogic Server 14.1.1.0.0.
+   * The existing WebLogic Server image, `container-registry.oracle.com/middleware/weblogic:12.2.1.3`,
+   has all the necessary patches applied.
+   * Check the WLS version with `docker run container-registry.oracle.com/middleware/weblogic:12.2.1.3 sh -c` `'source $ORACLE_HOME/wlserver/server/bin/setWLSEnv.sh > /dev/null 2>&1 && java weblogic.version'`.
+   * Check the WLS patches with `docker run container-registry.oracle.com/middleware/weblogic:12.2.1.3 sh -c` `'$ORACLE_HOME/OPatch/opatch lspatches'`.
+* Container images based on Oracle Linux 8 are now supported. The Oracle Container Registry hosts container images
+  based on both Oracle Linux 7 and 8, including Oracle WebLogic Server 14.1.1.0.0 images based on Java 8 and 11.
+* You must have the `cluster-admin` role to install the operator.  The operator does
+  not need the `cluster-admin` role at runtime.
+* We do not currently support running WebLogic in non-Linux containers.
+
+
+### Operator image
```

Comment:
This 'operator image' information seems out of place here to me. I assume it's already mentioned in the samples & the operator user guide.  

@rjeberhard thoughts?

Reply:
I'm okay with running this here. If I remember correctly, this was added because a specific customer was asking about running on MSFT OS images.

Reply:
Ok.

Reply:
This statement doesn't say anything about MSFT images: it's about getting the operator image, but this page is now solely about pre-requisites. IMO, the information better fits in the operator setup/install doc - I assume folks would naturally look for it there.

Reply:
Sorry, but I don't think that it's a stretch to have the operator image location on the prerequisites page. We can certainly include it in the Setup checklist. The Install the Operator doc is about installing with the operator Helm chart, https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-operators/installation/

Reply:
It still seems pretty odd to me. This page is about pre-reqs needed _before_ installing WKO. If we're going to mention a location for getting an operator image, why are we mentioning here, where no one would think to look? And if it belongs here, then why not also include any of the other stuff involved in installing and running the operator or running WKO itself?

Reply:
ok, I see your point. It should definitely be in the Setup doc. Where else? Do you see it fitting in the Installing the Operator doc?

Reply:
IMO operator install/setup information should be coalesced into the operator UG section, and general setup guidance should point to the operator UG section as needed.   In turn these sections should also reference the main sample(s) we have in the area.